tests/test_ntp.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import re
+
+
+@pytest.fixture()
+def GetTimeServers(File):
+    """ Extract the timeservers from ntp.conf
+
+    Args:
+        File - Module to access ntp config file
+
+    Returns:
+        ntp_servers - List of configured servers as indicated by ntp.conf
+    """
+    ntp_conf = File("/etc/ntp.conf")
+
+    # Filter fallback NTP server from configuration file.
+    ntp_servers = filter(lambda x: x != 'ntp.ubuntu.com',
+                         re.findall(
+                            "\n(?:server|pool)\s(.*)",
+                            ntp_conf.content_string))
+    return ntp_servers
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    """ Load default variables into dictionary.
+
+    Args:
+        Ansible - Requires the ansible connection backend.
+    """
+    return Ansible("include_vars", "./defaults/main.yml")["ansible_facts"]
+
+
+def test_ntp_exists(Package, AnsibleDefaults, GetTimeServers):
+    """ Ensure the desired version of NTP is installed.
+
+    Args:
+        Package - Module to determine package install status and version
+        AnsibleDefaults - Get desired package version
+    """
+    ntp = Package("ntp")
+    ntp_target_version = AnsibleDefaults["ntp_version"].split("*")[0]
+    assert ntp.is_installed
+    assert ntp.version.startswith(ntp_target_version)
+
+
+def test_ntp_config(AnsibleDefaults, GetTimeServers):
+    """ Ensure NTP is configured to use the proper timeservers
+
+    Args:
+        AnsibleDefaults - Get desired timeservers
+        GetTimeServers - Get configured timeservers
+    """
+    ntp_target_servers = AnsibleDefaults["ntp_servers"]
+    assert(set(GetTimeServers) == set(ntp_target_servers))
```

Comment:
In order for two lists to be equal, the order has to be the same. The `Set` class doesn't doesn't have that requirement (each set just has to be a subset of the other), so I thought it'd be cleaner to cast them as sets rather than check if each element in `GetTimeServers` is in `ntp_target_servers`.